VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/consultation-modals/consultation-leave/consultation-leave.component.ts

```diff
@@ -1,15 +1,20 @@
-import { Component, EventEmitter, Output } from '@angular/core';
+import { AfterViewInit, Component, EventEmitter, Output } from '@angular/core';
+import { ModalTrapFocus } from '../../../shared/modal/modal-trap-focus';
 @Component({
     selector: 'app-consultation-leave',
     templateUrl: './consultation-leave.component.html',
     styleUrls: ['./consultation-leave.component.scss']
 })
-export class ConsultationLeaveComponent {
+export class ConsultationLeaveComponent implements AfterViewInit {
     @Output() closedModal = new EventEmitter();
     @Output() leave = new EventEmitter();
 
     constructor() {}
 
+    ngAfterViewInit(): void {
+        ModalTrapFocus.trap('modal-window-confirmation');
```

Comment:
As per previous comment, can we use a ViewChild instead?

Reply:
created constants